cmd/vsphere-cloud-controller-manager/main.go

```diff
@@ -65,6 +65,24 @@ func main() {
 		Use: "cloud-controller-manager",
 		Long: `The Cloud controller manager is a daemon that embeds
 the cloud specific control loops shipped with Kubernetes.`,
+		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			// Glog requires this otherwise it complains.
+			goflag.CommandLine.Parse(nil)
+
+			// This is a temporary hack to enable proper logging until upstream dependencies
+			// are migrated to fully utilize klog instead of glog.
+			klogFlags := goflag.NewFlagSet("klog", goflag.ExitOnError)
```

Comment:
Is it just k8s.io/kubernetes using glog at the moment? If/when we update k/k vendor to use a version that is using klog can we remove this?

Reply:
Definitely k/k, but not really sure what else. We can definitely take a look at it when we vendor in a new version.

Reply:
Resolve based on the discussion in slack:

> andrewsykim: There are some optimizations and bug fixes that went in from v1.11 so we should have a discussion about what version to go to before GA though, but yeah feel free to hold cancel that one, maybe add a TODO comment